src/cmds/scripts/pbs_habitat.in

```diff
@@ -676,7 +675,10 @@ if [ "${PBS_START_SERVER:-0}" != 0 ] ; then
 			if [ -d "${old_inst_dir}" ]; then
 				backupdir "$old_inst_dir" "$PBS_HOME"
 				if [ $? -ne 0 ]; then
-					echo "*** ${old_inst_dir} need to be manually backup and copy ${PBS_EXEC}/pgsql as ${PBS_HOME}/pgsql.forupgrade."
+					echo "*** ${old_inst_dir} needs to be manually backed up and run the below command"
```

Comment:
nit picking - 
"and run the below" should be "by running the below command"

Reply:
It should be "and run the below".
Because if above backupdir function fails, ${PBS_HOME}/pgsql.forupgrade will remain exists and command at [line#850](https://github.com/PBSPro/pbspro/pull/1136/files#diff-ef7b0a6a28f4395057f99c8323e91a7fR850) will not be executed. Hence this message tells the user to take backup manually and run the given command, which does the same what the [line#850](https://github.com/PBSPro/pbspro/pull/1136/files#diff-ef7b0a6a28f4395057f99c8323e91a7fR850) would do on successful backup.

Reply:
Okay, thank you for the explanation offline.
As discussed we can do the following - 
If backupdir() fails - 
echo "failed to backup "$old_inst_dir""
echo "*** Kindly backup "$old_inst_dir" if you need to downgrade pgsql later on.
echo "*** For future upgrades to be successful run the below command."
echo "*** cp -pr ${PBS_EXEC}/pgsql ${PBS_HOME}/pgsql.forupgrade"